about.html

```diff
@@ -41,58 +45,37 @@ <h1 class="sr-only">If These Lands Could Talk</h1>
 							<button
 								id="mobile-menu"
 								class="mobile-menu"
-								aria-label="toggle mobile menu"
+								aria-label="toggle mobile menu on"
 							>
 								<!-- Hamburger Icon -->
-								<svg
-									width="24"
-									height="19"
-									viewBox="0 0 24 19"
-									xmlns="http://www.w3.org/2000/svg"
+								<img
+									src="/assets/icons/global/menu-icon.svg"
+									alt="mobile menu icon"
 									class="menu-icon"
-								>
-									<path
-										d="M22.3465 0H1.65354C1.215 0 0.794411 0.174212 0.484311 0.484311C0.174212 0.794411 0 1.215 0 1.65354C0 2.09209 0.174212 2.51267 0.484311 2.82277C0.794411 3.13287 1.215 3.30709 1.65354 3.30709H22.3465C22.785 3.30709 23.2056 3.13287 23.5157 2.82277C23.8258 2.51267 24 2.09209 24 1.65354C24 1.215 23.8258 0.794411 23.5157 0.484311C23.2056 0.174212 22.785 0 22.3465 0Z"
-									/>
-									<path
-										d="M22.3465 7.79492H1.65354C1.215 7.79492 0.794411 7.96913 0.484311 8.27923C0.174212 8.58933 0 9.00992 0 9.44847C0 9.88701 0.174212 10.3076 0.484311 10.6177C0.794411 10.9278 1.215 11.102 1.65354 11.102H22.3465C22.785 11.102 23.2056 10.9278 23.5157 10.6177C23.8258 10.3076 24 9.88701 24 9.44847C24 9.00992 23.8258 8.58933 23.5157 8.27923C23.2056 7.96913 22.785 7.79492 22.3465 7.79492Z"
-									/>
-									<path
-										d="M24 17.2444C24 16.8058 23.8258 16.3852 23.5157 16.0751C23.2056 15.765 22.785 15.5908 22.3465 15.5908H1.65354C1.215 15.5908 0.794411 15.765 0.484311 16.0751C0.174212 16.3852 0 16.8058 0 17.2444C0 17.6829 0.174212 18.1035 0.484311 18.4136C0.794411 18.7237 1.215 18.8979 1.65354 18.8979H22.3465C22.785 18.8979 23.2056 18.7237 23.5157 18.4136C23.8258 18.1035 24 17.6829 24 17.2444Z"
-									/>
-								</svg>
+								/>
 								<!-- exit menu icon -->
-								<svg
-									width="24"
-									height="24"
-									viewBox="0 0 24 24"
-									xmlns="http://www.w3.org/2000/svg"
+								<img
```

Comment:
Now using img src instead of inline svg

Reply:
Inline SVGs always feel like  a good idea at the time 🐶

Reply:
if only we were working with functional components :')